Lib/test/test_binascii.py

```diff
@@ -416,6 +423,21 @@ def test_b2a_base64_newline(self):
         self.assertEqual(binascii.b2a_base64(b, newline=False),
                          b'aGVsbG8=')
 
+    def test_deprecated_warnings(self):
+        with support.check_warnings(('', DeprecationWarning)):
```

Comment:
It is better to use standard `assertWarns()`.

Reply:
Done